ethcore/wasm/run/src/fixture.rs

```diff
@@ -41,8 +41,8 @@ pub struct Fixture {
 
 #[derive(Deserialize, Debug)]
 pub struct StorageEntry {
-	pub key: Uint,
```

Comment:
why? uint provide more flexibility, you don't need to write `0x0000000000000000..00000000` for `0x`

Reply:
but with explicit H256 you’ll never mess up with endianess on converting to bytes